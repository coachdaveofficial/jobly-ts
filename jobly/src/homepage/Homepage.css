.homepage {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    height: 100vh; /* Set the height to the full viewport height */
    background-image: url("../milky-way-2695569_1280.jpg");
  }
  
  .homepage-title {
    font-size: 4rem;
    font-weight: bold;
    color: #ffffff;
    text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5);
  }
  
  .homepage-subtitle {
    font-size: 2rem;
    color: #ffffff;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
  }
  