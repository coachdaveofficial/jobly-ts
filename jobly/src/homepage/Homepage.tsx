import React, {useContext} from 'react'



export default function Homepage() {
    return (
        <div>
            hello homepage
        </div>
    )
}