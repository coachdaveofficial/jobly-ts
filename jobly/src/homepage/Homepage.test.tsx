import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import Homepage from "./Homepage";

it("matches the snapshot", () => {
    const {asFragment} = render(
        <MemoryRouter>
            <Homepage />
        </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
});

it("renders without crashing", () => {
    const {getByText} = render(<Homepage />);

    expect(getByText("Jobly")).toBeInTheDocument();
})