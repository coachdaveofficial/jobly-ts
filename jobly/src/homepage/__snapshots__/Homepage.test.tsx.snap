// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="homepage"
  >
    <h1
      class="homepage-title"
    >
      Jobly
    </h1>
    <p
      class="homepage-subtitle"
    >
      A job board app
    </p>
  </div>
</DocumentFragment>
`;
